source/Mission.h

```diff
@@ -15,6 +15,7 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
 #include "ConditionSet.h"
 #include "Date.h"
+#include "DistanceMap.h"
```

Comment:
This should not be in the Mission class header. Split the enum to its own file and include it in both instead.

Reply:
Done, I think? Named it `wormholestrategy.hpp` based on `alignment.hpp`.